nlpsandboxclient/cli/community.py

```diff
@@ -25,55 +25,56 @@ def get_num_users():
 @click.option('--data_node_host',
               help='Data node host. If not specified, uses '
                    'http://0.0.0.0:8080/api/v1')
```

Comment:
Should be replaced by a variable that is set to the default value to avoid code duplication.